transmartApp/src/main/groovy/org/transmart/FederatedUserDetailsService.groovy

```diff
@@ -1,6 +1,5 @@
 package org.transmart
```

Comment:
This class (and SAML authentication in general) should probably also be moved to transmart-oauth. But it would be difficult to test if it still works...